locale/en-us.yaml

```diff
@@ -631,9 +631,11 @@ secrets_header_description:
 secrets_header_usage:
   other: Usage
 secrets_row_value_set:
-  other: Defined
+  other: "[SUCCESS]Defined[/RESET]"
 secrets_row_value_unset:
-  other: Undefined
+  other: "[ERROR]Not defined[/RESET]"
```

Comment:
These are different from the mock as the output was empty with the special characters. Filed a story here: https://www.pivotaltracker.com/story/show/176372179